lib/fluent/plugin/in_prometheus.rb

```diff
@@ -11,6 +12,7 @@ class PrometheusInput < Fluent::Plugin::Input
     config_param :bind, :string, default: '0.0.0.0'
     config_param :port, :integer, default: 24231
     config_param :metrics_path, :string, default: '/metrics'
+    config_param :metrics_all_path, :string, default: '/metrics_all'
```

Comment:
How about `aggregated_metrics` ?

Reply:
Sure, done in 3eff52fd5dd63c1cb429a2a666fa7bdb2fde48e2